src/svd.jl

```diff
@@ -245,6 +245,37 @@ svdvals(A::AbstractVector{T}) where {T} = [convert(eigtype(T), norm(A))]
 svdvals(x::Number) = abs(x)
 svdvals(S::SVD{<:Any,T}) where {T} = (S.S)::Vector{T}
 
+"""
+    rank(S::SVD{<:Any, T}; atol::Real=0, rtol::Real=min(n,m)*ϵ) where {T}
```

Comment:
I don't think you meant to include this in this PR?

Reply:
this was a mistake sorry.

Reply:
I am new to this. How do I delete changes from a PR?